plutus-contract/test/Spec/Contract/TxConstraints.hs

```diff
@@ -0,0 +1,299 @@
+{-# LANGUAGE BlockArguments        #-}
```

Comment:
Why was this file added? Seems like an old file that was added back.. Am I wrong?

Reply:
I guess I didn't bay enough attention during the rebase (I probably changed something to it and git thought I wanted to keep it while James removed it).